src/listpack.c

```diff
@@ -45,98 +45,6 @@
 #include "serverassert.h"
 #include "util.h"
 
-#define LP_HDR_SIZE 6       /* 32 bit total len + 16 bit number of elements. */
```

Comment:
I'm a little hesitant about exporting all this, since it complicates the header file a bit. I think I would rather use the alternative testing strategy we have, which is to `#include "../listpack.c"` in the test file, similar to https://github.com/valkey-io/valkey/blob/c7ad9feb52973da8ba7d49b5530837466a910477/src/unit/test_intset.c#L4.